helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -274,13 +274,8 @@ private Response batchGetStoppableInstances(String clusterId, JsonNode node, boo
       return JSONRepresentation(result);
     } catch (HelixException e) {
       _logger
-          .error(String.format("Current cluster %s has issue with health checks!", clusterId), e);
+              .error(String.format("Current cluster %s has issue with health checks!", clusterId), e);
       throw new HelixHealthException(e);
-    } catch (Exception e) {
-      _logger.error(String.format(
-              "Failed to get parallel stoppable instances for cluster %s with a HelixException!",
-              clusterId), e);
```

Comment:
Let's keep the log here? Logging the cluster name will be helpful when we debug.